scripts/autotools/libmongocrypt/Version.m4

```diff
@@ -0,0 +1,6 @@
+if test "$PHP_CLIENT_SIDE_ENCRYPTION" != "no"; then
```

Comment:
Should this be changed to "yes" (as other conditionals in `config.m4`), or is it safer to leave as-is so `Version.m4` does not subtly depend on being included _after_ `CheckSSL.m4`? I'm OK with either, and there's certainly no harm in doing the `AC_SUBST` for both the "yes" and "auto" case if we did happen to include this before `CheckSSL.m4`.

Reply:
I've changed this to yes: we should first check if we can compile with libmongocrypt, then handle all other libmongocrypt tasks.